pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -240,7 +240,21 @@ qtModule {
   meta = with lib; {
     description = "A web engine based on the Chromium web browser";
     maintainers = with maintainers; [ matthewbauer ];
-    platforms = platforms.unix;
+
+    # qtwebengine-5.15.8: "QtWebEngine can only be built for x86, x86-64, ARM, Aarch64, and MIPSel architectures."
```

Comment:
Split the comment in the classic 80-char limit.

Reply:
Done.

Reply:
FYI We don't care about 80 chars. 120 or 150 are also fine.

Reply:
There is/was a slide bar in the web version of github.